### ISSUE TITLE
devDependencies packages are saved as dependencies when using wizard

### ISSUE THREAD

**Issue by getglad** [Snyk/snyk#23]
- `node -v`: 4.4.3
- `npm -v`: 3.8.6
- `snyk -v`: 1.14.0

- Command run: snyk wizard --dev

### Expected behaviour
It would update/save any fixes to a package under `devDependencies` to the `devDependencies` section of package.json

So if package.json reads:
```
"dependencies": {},
"devDependencies": {
   gulp-uglify: 1.4.1
}
```
After running, it will read:
```
"dependencies": {},
"devDependencies": {
   gulp-uglify: 1.5.1
}
```

### Actual behaviour
Any updated packages in `devDependencies` are added to  `dependencies`

Before:
```
"dependencies": {},
"devDependencies": {
   gulp-uglify: 1.4.1
}
```
After:
```
"dependencies": {
   gulp-uglify: 1.5.1
},
"devDependencies": {
   gulp-uglify: 1.4.1
}
```

### Steps to reproduce
Run the wizard command with the `--dev` flag. Examine changes to `devDependencies` vs `dependencies`

**Comment by remy** [comment]
Ah, good catch! I know why this is happening, and will require an upgrade in the wizard, but definitely a bug.

I'll report back with progress, thanks again!

**Comment by getglad** [comment]
Cool - this is a great tool btw

**Comment by remy** [comment]
This should be fixed and live (finally) in snyk@1.17.0. 

Thank you so much for filing and being patient! :)
